examples/example3/soultz_grid.py

```diff
@@ -7,7 +7,7 @@
 from porepy.fracs import meshing, simplex, split_grid
```

Comment:
With recent updates to the fracture and fractureNetwork classes, I think we can simply mesh this data set with no modifications. Just leave your modifications for now, then I will simplify later. However, we should keep in mind reproducibility of your OGST results.